### ISSUE TITLE
partial: additional indefinite article in error message

### ISSUE THREAD

**Issue by shosaco** [tidyverse/purrr#696]
There are additional indefinite articles in some error messages. This happens because `friendly_type_of` outputs the indefinite article *a* (e.g. *a character vector*) and `partial` also outputs *a friendly_type_of(.f)*, resulting in double *a* in its error message - e.g. *a a character vector*.

``` r
library(purrr)
test_path <- "C:"
partial(file.path(test_path))
#> `.f` must be a function, not a a character vector
```

**Comment by shosaco** [comment]
fixed in Pull request  #697

**Comment by lionel-** [comment]
Thanks for the fix.
